src/services/commands.js

```diff
@@ -9,7 +9,16 @@ Controller.open(function(_) {
       ignoredCharacters: {}
     };
 
-    // Process injected commands into autocommands 
+    // MatHSPaCE HacK - Allow multiplication sign to be configurable
+    var multiplicationDisplaySymbol =
```

Comment:
What's the point in mutating `this.cursor.options.multiplicationDisplaySymbol` during latex grammar initialization? Seems a weird thing to do.

Do we even need to define a local variable `multiplicationDisplaySymbol`?

Why not just this:

```js
if (this.cursor.options.multiplicationDisplaySymbol === "dot") {
  this.cursor.grammarDicts.latexCmds['*'] = LatexCmds.cdot;
}
```

`this.cursor.options.multiplicationDisplaySymbol === "dot"` will only be true if you've explicitly passed option with `{ multiplicationDisplaySymbol: true }`; will be false otherwise.

Reply:
Honestly, I was just doing the same code style as the other options a couple lines above. But you're right, it makes no sense and we can simplify it.